caikit_nlp/toolkit/verbalizer_utils.py

```diff
@@ -89,15 +89,19 @@ def replace_text(match_obj):
             if index_object not in source_object:
                 error(
                     "<NLP97415192E>",
-                    KeyError("Requested template string is not a valid key in dict"),
+                    KeyError(
+                        "Requested template string '%s' is not a valid key in dict",
+                        index_object,
+                    ),
```

Comment:
nit: can we change it to inline f string format:

```suggestion
                    KeyError(
                        f"Requested template string '{index_object}' is not a valid key in dict"
                    ),
```